WordPress/src/main/java/org/wordpress/android/ui/people/PersonDetailFragment.java

```diff
@@ -105,11 +118,31 @@ public void refreshPersonDetails() {
             String avatarUrl = GravatarUtils.fixGravatarUrl(person.getAvatarUrl(), avatarSz);
 
             mAvatarImageView.setImageUrl(avatarUrl, WPNetworkImageView.ImageType.AVATAR);
-            mDisplayNameTextView.setText(person.getDisplayName());
-            mUsernameTextView.setText(person.getUsername());
+            mDisplayNameTextView.setText(StringUtils.unescapeHTML(person.getDisplayName()));
             mRoleTextView.setText(StringUtils.capitalize(person.getRole()));
 
-            setupRoleContainerForCapability();
+            if (!person.getUsername().isEmpty()) {
```

Comment:
It would be better to use `TextUtils.isEmpty(person.getUsername())` to prevent a possible NPE from happening